### PR TITLE
Page component continues

### PR DESCRIPTION
**Pull request by amiryonatan**
## Before you request a review, please make sure to check the following boxes:

- [ ] Explain what has changed.
- [ ] Explain why the change is needed.
- [ ] Added tests.
- [ ] Update documentation.
- [ ] Merge master with no conflicts.
- [ ] Make sure the build is passing on ci.
- [ ] If this pr closes any issue, please add a github link to close it.
- [ ] If you changed something in the product design/UX, you got the approval of [Ben Benhorin](https://wix.slack.com/messages/@benb).

### CODE REVIEW

File: src/Page/PageHeader/PageHeader.js [hunk 1]
```diff
@@ -31,40 +32,46 @@ export const generateDefaultBreadcrumbs = title =>
...
-            )
-          }
-          {subtitle &&
-            animateComponent(!minimized, !breadcrumbs,
-              <div className={classNames(s.subtitle, {[s.minimized]: minimized})} data-hook="page-header-subtitle">
-                <Text appearance="T1.1">{subtitle}</Text>
-              </div>
-            )
+            backCallback && animateComponent(!minimized, !breadcrumbs,
+              <div onClick={() => backCallback()} className={classNames(s.backButton, {[s.minimized]: minimized})}>
```
Comment (shlomitc) [inline src/Page/PageHeader/PageHeader.js]: pass the event as well `e => backCallback(e)`

File: src/Page/PageHeader/PageHeader.js [hunk 2]
```diff
@@ -31,40 +32,46 @@ export const generateDefaultBreadcrumbs = title =>
   */
 export class PageHeader extends WixComponent {
   render() {
-    const {breadcrumbs, showBackButton, title, subtitle, minimized} = this.props;
+    const {breadcrumbs, backCallback, title, subtitle, minimized, actionsBar} = this.props;
```
Comment (shlomitc) [inline src/Page/PageHeader/PageHeader.js]: I prefer explicitly showing the back button or not and add a callback in addition.
Also, rename `backCallback to `onBackClicked` or something similar.

File: src/Page/PageHeader/PageHeader.js [hunk 3]
```diff
@@ -31,40 +32,46 @@ export const generateDefaultBreadcrumbs = title =>
...
-              </div>
-            )
-          }
-          {subtitle &&
-            animateComponent(!minimized, !breadcrumbs,
-              <div className={classNames(s.subtitle, {[s.minimized]: minimized})} data-hook="page-header-subtitle">
-                <Text appearance="T1.1">{subtitle}</Text>
-              </div>
-            )
+            backCallback && animateComponent(!minimized, !breadcrumbs,
```
Comment (shlomitc) [inline src/Page/PageHeader/PageHeader.js]: show based on a specific prop and not the callback

File: src/Page/PageHeader/PageHeader.js (hunk 2 above)
Comment (amiryonatan) [reply src/Page/PageHeader/PageHeader.js]: 👍

File: src/Page/PageHeader/PageHeader.js (hunk 3 above)
Comment (amiryonatan) [reply src/Page/PageHeader/PageHeader.js]: 👍

File: src/Page/PageHeader/PageHeader.js (hunk 1 above)
Comment (amiryonatan) [reply src/Page/PageHeader/PageHeader.js]: 👍

**Pull request merged**